### PR TITLE
Workout page backend

### PR DESCRIPTION
**Pull request by KalesmatBBD**

### CODE REVIEW

File: models/Server.js [hunk 1]
```diff
@@ -2,460 +2,503 @@ var sql = require("mssql");
 const date = require("date-and-time");
 
 var config = {
-  user: "root",
```
Comment (JordanFalkBBD) [inline models/Server.js:5]: Why are you changing the connection string?

Comment (KalesmatBBD) [reply models/Server.js:5]: Sorry forgot to put it back when I was testing I have changed it back

File: models/Server.js [hunk 2]
```diff
@@ -2,460 +2,503 @@ var sql = require("mssql");
...
-  password: "FitnessApp",
-  server: "mssqldb.cjovnczdjuek.eu-west-1.rds.amazonaws.com",
-  database: "FitnessAppDB",
-  trustServerCertificate: true,
+    user: 'sa',
```
Comment (JordanFalkBBD) [inline models/Server.js]: It must be "root"

File: models/Server.js [hunk 3]
```diff
@@ -2,460 +2,503 @@ var sql = require("mssql");
...
+        await sql.close();
+    
+        return updateWorkout;
+      } catch (error) {
+        console.error('Error:', error.message);
+        throw error;
+      }
 }
 
-async function fetchExercisesForUser(userID) {
```
Comment (JordanFalkBBD) [inline models/Server.js]: Why are you removing these functions?

File: models/Server.js [hunk 4]
```diff
@@ -2,460 +2,503 @@ var sql = require("mssql");
...
-    await sql.close();
-
-    return exercises;
-  } catch (error) {
-    console.error("Error:", error.message);
-    throw error;
-  }
-}
-
-async function fetchCardioForUser(userID) {
```
Comment (JordanFalkBBD) [inline models/Server.js]: This one too

**Review by JordanFalkBBD — APPROVED** [review]

**Pull request merged**